spec/support/shared_examples/crud.rb

```diff
@@ -128,6 +128,21 @@
         end
       end
     end
+
+    if methods.include? :search
+      context '::search' do
+        let(:status) { 200 }
+        let(:response) { { described_class.plural_key => [{}] } }
+        let(:params) { property ? { property.to_sym => '13666', first_name: "John" } : {} }
```

Comment:
Line is too long. [91/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.